src/Workspaces/Core/Portable/Execution/AbstractReferenceSerializationService.cs

```diff
@@ -159,6 +206,50 @@ public AnalyzerReference ReadAnalyzerReferenceFrom(ObjectReader reader, Cancella
             throw ExceptionUtilities.UnexpectedValue(type);
         }
 
+        private Checksum CreateAnalyzerReferenceChecksum(AnalyzerReference reference, CancellationToken cancellationToken)
```

Comment:
What is the purpose of introducing CreateAnalyzerReferenceChecksum? The original CreateChecksum method becomes an almost transparent. Does CreateChecksum still add a value?